### PR TITLE
add any version env #32

### PR DESCRIPTION
**Pull request by btnoguchimnm**
please Review #32 .

### CODE REVIEW

**Comment by btnoguchimnm** [comment]
Since the archetype( 5.1.2-SNAPSHOT , 5.2.1-SNAPSHOT , 5.3.0-SNAPSHOT ) has not been made, this PR is error has occurred.
please review(and marge) when the archetype is made and the error has not occorred .

**Comment by ikeyat** [comment]
@btnoguchimnm 
I re-run travis ci script just now.
The errors may be occurred because the versions in the github master branch are not updated in that time.
Now we've updated, so tests will success.

File: first-spring-security/scripts/convert-security-xml.sh [hunk 1]
```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
 # first-spring-security-mybatis3-env.xml
+if test ${ARCHETYPE_VERSION:0:5} = "5.1.0" -o ${ARCHETYPE_VERSION:0:5} = "5.1.1" -o ${ARCHETYPE_VERSION:0:5} = "5.2.0";then
```
Comment (btyoshikawaa) [inline first-spring-security/scripts/convert-security-xml.sh]: - [ ] It seems to be better to use case statements.

Comment (btnoguchimnm) [inline first-spring-security/scripts/convert-security-xml.sh]: [daf686f]
I corresponded .

**Pull request merged**